Tasks/WindowsMachineFileCopy/README.md

```diff
@@ -5,7 +5,7 @@ The task is used to copy application files and other artifacts that are required
 ###The different parameters of the task are explained below:
 
 *	**Source**: The source of the files. As described above using pre-defined system variables like $(Build.Repository.LocalPath) make it easy to specify the location of the build on the Build Automation Agent machine. The variables resolve to the working folder on the agent machine, when the task is run on it. Wild cards like **\*.zip are not supported.
-* **Machines**: Specify comma separated list of machine FQDNs/ip addresses along with port(optional). For example dbserver.fabrikam.com, dbserver_int.fabrikam.com:5986,192.168.34:5986. Port when not specified will be defaulted to WinRM defaults based on the specified protocol. i.e., (For *WinRM 2.0*):  The default HTTP port is 5985, and the default HTTPS port is 5986. Machines field also accepts 'Machine Groups' defined under 'Test' hub, 'Machines' tab. 
+* **Machines**: Specify comma separated list of machine FQDNs/ip addresses along with port(optional). For example dbserver.fabrikam.com, dbserver_int.fabrikam.com:5986,192.168.34:5986. Machines field also accepts 'Machine Groups' defined under 'Test' hub, 'Machines' tab. 
```

Comment:
we can discuss further offline. and that change (if coming) can be incremental. I am going ahead and merging this